q2_cutadapt/tests/test_trim.py

```diff
@@ -156,7 +156,8 @@ def test_build_trim_command_typical(self):
                                       overlap=4,
                                       match_read_wildcards=True,
                                       match_adapter_wildcards=False,
-                                      minimum_length=2)
+                                      minimum_length=2,
+                                      discard_untrimmed=True)
```

Comment:
here and below: should we be making sure that this is working as intended? i.e., that specific IDs are dropped/retained (or at least a specific # of seqs is retained). I get it if you don't feel we should be in the business of validating cutadapt...

Reply:
My thinking here is test that we are writing the command out correctly, leave the implementation up to cutadapt. However, given that this whole problem came out of the "empty record" problem, I should probably add at least one test for _that case_. Thanks!